### DISCUSSION TITLE
Seminar Discussions

### DISCUSSION THREAD

**Discussion by mariotamsc5** [mariotamsc5/Seminar#1] [Announcements]
We can use the discussion part to choose on what each of us is going to work on, to share ideas and to ask possible questions that we may have.

**Comment by Qaswara98** [comment]
Maria and I have been working on creating an R package with documentation using RStudio. We’ve successfully pushed it to this repository. The package is currently named BioEnrichR. However, we’re open to suggestions for a new name. If anyone has a better name in mind, feel free to share it here so we can discuss it. Thank you! 😊
